internal/generator/generator.go

```diff
@@ -9,10 +9,10 @@ import (
 	"github.com/sunboyy/repogen/internal/spec"
 )
 
-func GenerateRepositoryImpl(pkg *types.Package, structModelName,
-	repoInterfaceName string) (string, error) {
+func GenerateRepositoryImpl(pkg, modelPkg *types.Package, structModelName,
```

Comment:
Rename `pkg` to `repoPkg` and swap modelPkg to come before repoPkg for consistency